code-challenges.js

```diff
@@ -3,18 +3,34 @@
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
-
+const fibTen = (n) => {
+    if (n === 1) {
+        return [0] 
+    } else if (n === 2){
+        return [1, 1]
+    } else {
+        var ansArr = fibTen(n - 1)
+         ansArr.push(ansArr[ansArr.length -1] + ansArr[ansArr.length-2])
+         return ansArr 
+    }
+}
+console.log(fibTen(10))
```

Comment:
cool recursion 😎. just make sure you comment your code since it's not immediately obvious what you're doing here.